serverless.yml

```diff
@@ -0,0 +1,50 @@
+service: bref-demo-symfony
+
+provider:
+  name: aws
+  region: eu-west-1
+  runtime: provided
+  environment:
+    # Symfony environment variables
+    APP_ENV: dev
```

Comment:
this file is used in dev or prod?

Reply:
committed values are "reasonable defaults" for dev ... on prod they are ment to be replaced by proper ENV vars

Reply:
👍